WordPress/Classes/Models/Blog+Post.swift

```diff
@@ -28,3 +30,82 @@ public extension Blog {
         return (try? context.fetch(request))?.first
     }
 }
+
+// MARK: - Create posts
+
+extension Blog {
+
+    /// Create a post in the blog.
+    @objc
+    func createPost() -> Post {
+        guard let context = managedObjectContext else {
+            fatalError("The `Blog` instance is not associated with a `NSManagedObjectContext`")
+        }
+
+        let post = NSEntityDescription.insertNewObject(forEntityName: NSStringFromClass(Post.self), into: context) as! Post
+        post.blog = self
+        post.remoteStatus = .sync
+
+        if let categoryID = settings?.defaultCategoryID,
+            categoryID.intValue != PostCategoryUncategorized,
```

Comment:
I verified the indentation using Xcode, and it applied this change:

```suggestion
           categoryID.intValue != PostCategoryUncategorized,
```

Reply:
🥲 Good catch!